.github/PULL_REQUEST_TEMPLATE/stamp_pull_request_template.md

```diff
@@ -0,0 +1,52 @@
+<!--
+Instructions:
+
+Hello! Thank you for taking the time to propose a stamp for the Passport app.
+We're excited to work with you on reviewing your stamp. Please fill out the
+following information. Failure to do so will result in delays reviewing your
+stamp or we may reject your proposal entirely.
+
+You'll find a series of questions and fields below containing information we
+need from you to properly review your stamp. Any text surrounded by angle
+brackets (<>) should be replaced with the appropriate information about your
+project. Headings should be followed by a few sentences that answer the question
+or explain the topic.
+
+When you finish, please fill out this Google form: https://forms.gle/29aqP3shRivRR6Bo6
+
+-->
+# <Stamp Provider>
+<!--
+Replace this comment with a brief (1-3 sentences) paragraph explaining your
+project.
+-->
+
+## Meta Data
+
+Project Name: <project name>
+
+Website: <https://www.project.com>
+
+Daily Active Users (DAUs): <420,000>
+
+Stamp name: <stamp name>
+
+## Data and Credentials
+
+<!--
+For a stamp to be usefull, it should include a couple of credentials. For
+each verifiable credential you issue as part of your stamp, please provide:
+
+- the vc name (i.e. `gtcPosessionsGte#10`)
+- the data point in your app that supports this VC (i.e. )
```

Comment:
You might still be working on this, but calling it out just in case. i.e. is empty.

Also worthwhile mentioning that i.e. is not the most accessible:
https://insidegovuk.blog.gov.uk/2016/07/20/changes-to-the-style-guide-no-more-eg-and-ie-etc/